blog/2025-04-15-renames/index.mdx

```diff
@@ -0,0 +1,159 @@
+---
+slug: rename-object
+title: "We solved one of the biggest problems in object storage"
```

Comment:
Should we make the title about `rename`?

Reply:
This is intentionally clickbait

Reply:
Makes sense.